### PR TITLE
Cut down size of several fetch results to improve speed of online tests

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: sunpy/net/tests/test_fido.py [hunk 1]
```diff
@@ -230,19 +227,19 @@ def filter_queries(queries):
...
+    assert file == [f'{tmp_path}/aia_lev1_335a_2022_01_01t00_00_00_62z_image_lev1.fits']
 
 
 @pytest.mark.remote_data
 @skip_windows
 def test_path_read_only(tmp_path):
     results = Fido.search(
-        a.Time("2012/1/1", "2012/1/5"), a.Instrument.lyra)
-
+        a.Time("2012/1/1", "2012/1/1"), a.Instrument.lyra, a.Level.two
```
Comment (nabobalis) [inline sunpy/net/tests/test_fido.py:241]: To cut down the search result to 1

File: sunpy/net/tests/test_fido.py [hunk 2]
```diff
@@ -230,19 +227,19 @@ def filter_queries(queries):
 
 
 @pytest.mark.remote_data
-def test_path():
+def test_path(tmp_path):
     results = Fido.search(
-        a.Time("2012/1/1", "2012/1/5"), a.Instrument.lyra)
-
-    Fido.fetch(results, path="notapath/{file}")
+        a.Time("2022/1/1", "2022/1/1"), a.Instrument.aia)
```
Comment (nabobalis) [inline sunpy/net/tests/test_fido.py:232]: To cut down the search result to 1 file which is smaller than a lyra file.

File: sunpy/net/dataretriever/sources/lyra.py [hunk 3]
```diff
@@ -32,8 +32,18 @@ class LYRAClient(GenericClient):
...
     2016-01-02 00:00:00.000 2016-01-02 23:59:59.999       LYRA ...      ESA     3
     <BLANKLINE>
+    3 Results from the VSOClient:
+    Source: http://vso.stanford.edu/cgi-bin/search
+    Total estimated size: 2.914 Gbyte
+    <BLANKLINE>
+           Start Time       ...          Info Required
+                            ...
+    ----------------------- ... --------------------------------
+    2016-01-01 09:41:00.000 ... ...
```
Comment (nabobalis) [inline sunpy/net/dataretriever/sources/lyra.py:42]: The info required values seemed to change on each query so I ignored them.

File: sunpy/net/vso/table_response.py [hunk 4]
```diff
@@ -83,13 +83,12 @@ def from_zeep_response(cls, response, *, client, _sort=True):
                         # doesn't recognise as a unit, so fix it.
                         waveunit = value['waveunit']
                         waveunit = 'keV' if waveunit == 'kev' else waveunit
-
                         row["Wavelength"] = None
-                        if value['wavemin'] is not None and value['wavemax'] is not None:
+                        # VSO added "N/A" as potential value for wavemin and wavemax
+                        if value['wavemin'] not in [None, "N/A"] and value['wavemax'] not in [None, "N/A"]:
```
Comment (Cadair) [inline sunpy/net/vso/table_response.py:88]: I would like to get word from the VSO that this is a permanent change before we do this.

Comment (nabobalis) [reply sunpy/net/vso/table_response.py:88]: @ejm4567 What are your thoughts on this change?

Comment (ejm4567) [reply sunpy/net/vso/table_response.py:88]: I've put a temp fix in the VSO Core for Lyra so you shouldn't see that error in SunPy again. It turns out the N/A is what is being returned by the TAP query in wavelength_range for Lyra. Swap populates that field with actual wavelengths. I need to look in detail at what other fields Lyra may be populating wherein I can then map correct, accurate wavelengths for Lyra. The fix would ideally be in the VSO wherein the query results are first processed.

Comment (nabobalis) [reply sunpy/net/vso/table_response.py:88]: Thanks! So it should be safe for me to revert this now?

File: docs/tutorial/acquiring_data/fido.rst [hunk 5]
```diff
@@ -494,29 +484,39 @@ Here we'll just download the first file in the result::
...
 
     >>> downloaded_files = Fido.fetch(results, path='./{instrument}/{file}')  # doctest: +SKIP
 
-You can see the list of options that can be specified in path for all the files to be downloaded with ``results.path_format_keys``::
+You can see the list of options that can be specified in path for all the files to be downloaded with ``results.path_format_keys``:
+
+.. code-block:: python
 
     >>> sorted(results.path_format_keys()) # doctest: +REMOTE_DATA
-    ['end_time', 'extent_length', 'extent_type', 'extent_width', 'fileid', 'info', 'instrument', 'physobs', 'provider', 'size', 'source', 'start_time', 'wavelength', 'wavetype']
```
Comment (nabobalis) [inline docs/tutorial/acquiring_data/fido.rst:510]: Not sure if this was meant to go but locally this went.

File: docs/tutorial/acquiring_data/fido.rst [hunk 6]
```diff
@@ -281,86 +317,74 @@ As we have seen above the `~sunpy.net.fido_factory.UnifiedResponse` object conta
...
 
 The VSO client returns a lot of information about the records, so the first thing we can do is show only the columns we are interested in.
-We can inspect all the available column names in all the responses with the `~.UnifiedResponse.all_colnames` property::
+We can inspect all the available column names in all the responses with the `~.UnifiedResponse.all_colnames` property:
+
+.. code-block:: python
 
     >>> results.all_colnames  # doctest: +REMOTE_DATA
-    ['End Time', 'Extent Length', 'Extent Type', 'Extent Width', 'Info', 'Instrument', 'Physobs', 'Provider', 'Size', 'Source', 'Start Time', 'Wavelength', 'Wavetype', 'fileid']
+    ['End Time', 'Extent Length', 'Extent Type', 'Extent Width', 'Instrument', 'Physobs', 'Provider', 'Size', 'Source', 'Start Time', 'Wavelength', 'Wavetype', 'fileid']
```
Comment (nabobalis) [inline docs/tutorial/acquiring_data/fido.rst:377]: Info went away.

File: docs/tutorial/acquiring_data/fido.rst [hunk 7]
```diff
@@ -281,86 +317,74 @@ As we have seen above the `~sunpy.net.fido_factory.UnifiedResponse` object conta
...
 
-    >>> results = Fido.search(a.Time("2020/01/01", "2020/01/01 00:05"), a.Instrument.aia | a.Instrument.hmi)  # doctest: +REMOTE_DATA
+.. code-block:: python
+
+    >>> results = Fido.search(a.Time("2020/01/01", "2020/01/01 00:01"), a.Instrument.aia | a.Instrument.hmi)  # doctest: +REMOTE_DATA
     >>> results  # doctest: +REMOTE_DATA
     <sunpy.net.fido_factory.UnifiedResponse object at ...>
     Results from 2 Providers:
     <BLANKLINE>
-    201 Results from the VSOClient:
```
Comment (nabobalis) [inline docs/tutorial/acquiring_data/fido.rst:291]: I cut the query down.

**Comment by nabobalis** [comment]
Online fails are:

```
FAILED ../../.tox/py39-online/lib/python3.9/site-packages/sunpy/net/helio/parser.py::sunpy.net.helio.parser.endpoint_parser
FAILED ../../.tox/py39-online/lib/python3.9/site-packages/sunpy/net/helio/parser.py::sunpy.net.helio.parser.taverna_parser
FAILED ../../.tox/py39-online/lib/python3.9/site-packages/sunpy/net/helio/parser.py::sunpy.net.helio.parser.wsdl_retriever
```

File: docs/tutorial/acquiring_data/fido.rst [hunk 8]
```diff
@@ -218,7 +236,9 @@ This means that the results of a call to search can contain many sets of records
 The results of a search are represented in a `~sunpy.net.fido_factory.UnifiedResponse` object, which provides access to all the response tables and allows some operations to be performed on all the results at once.
 `~sunpy.net.fido_factory.UnifiedResponse` acts both like a two dimensional array, where the first dimension is the response index and the second index is the row index, and a dictionary where you can index the responses by the name of the client.
 
-For example, the following code returns a response containing LYRA data from the `~sunpy.net.dataretriever.LYRAClient`, and EVE data from the `~sunpy.net.vso.VSOClient`::
+For example, the following code returns a response containing LYRA data from the `~sunpy.net.dataretriever.LYRAClient`, and EVE data from the `~sunpy.net.vso.VSOClient`:
+
+.. code-block:: python
```
Comment (Cadair) [inline docs/tutorial/acquiring_data/fido.rst:241]: Why?

File: docs/tutorial/acquiring_data/fido.rst [hunk 9]
```diff
@@ -238,22 +258,37 @@ For example, the following code returns a response containing LYRA data from the
...
+    2012-01-02 00:00:00.000 2012-01-02 01:00:00.000    SDO ...    FULLDISK -0.00098
+    2012-01-02 00:00:00.000 2012-01-02 01:00:00.000    SDO ...    FULLDISK -0.00098
     Length = 50 rows
     <BLANKLINE>
     <BLANKLINE>
 
+If you then wanted to inspect just the LYRA data for the whole time range specified in the search, you would index this response to see just the results returned by the `~sunpy.net.dataretriever.LYRAClient`:
 
-If you then wanted to inspect just the LYRA data for the whole time range specified in the search, you would index this response to see just the results returned by the `~sunpy.net.dataretriever.LYRAClient`::
+.. code-block:: python
```
Comment (Cadair) [inline docs/tutorial/acquiring_data/fido.rst:291]: Why?

File: docs/tutorial/acquiring_data/fido.rst [hunk 10]
```diff
@@ -280,86 +317,74 @@ As we have seen above the `~sunpy.net.fido_factory.UnifiedResponse` object conta
...
 
     >>> results.all_colnames  # doctest: +REMOTE_DATA
-    ['End Time', 'Extent Length', 'Extent Type', 'Extent Width', 'Info', 'Instrument', 'Physobs', 'Provider', 'Size', 'Source', 'Start Time', 'Wavelength', 'Wavetype', 'fileid']
+    ['End Time', 'Extent Length', 'Extent Type', 'Extent Width', 'Instrument', 'Physobs', 'Provider', 'Size', 'Source', 'Start Time', 'Wavelength', 'Wavetype', 'fileid']
+
+And then we can pick which ones to see with the :meth:`~.UnifiedResponse.show` method:
 
-And then we can pick which ones to see with the :meth:`~.UnifiedResponse.show` method::
+.. code-block:: python
```
Comment (Cadair) [inline docs/tutorial/acquiring_data/fido.rst:381]: Why?

**Review by Cadair — APPROVED** [review]
Your extraneous diffs are once again a pain in my backside.

File: sunpy/net/dataretriever/sources/lyra.py (hunk 3 above)
Comment (Cadair) [reply sunpy/net/dataretriever/sources/lyra.py:42]: We shouldn't be printing that column at all, see my comment on #6887

File: docs/tutorial/acquiring_data/fido.rst [hunk 11]
```diff
@@ -9,14 +9,18 @@ This guide outlines how to search for and download data using the `~sunpy.net.Fi
 It therefore supplies a single, easy, and consistent way to obtain most forms of solar physics data.
 
 The `~sunpy.net.Fido` object is in `sunpy.net`.
-All the examples in this guide use ``Fido``, so lets start by importing it::
+All the examples in this guide use ``Fido``, so lets start by importing it:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:14]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 12]
```diff
@@ -9,14 +9,18 @@ This guide outlines how to search for and download data using the `~sunpy.net.Fi
...
 
     >>> from sunpy.net import Fido, attrs as a
 
 .. contents::
     :depth: 3
 
-Fido supports a number of different remote data sources. To see a list the Fido object can be printed::
+Fido supports a number of different remote data sources. To see a list the Fido object can be printed:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:23]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 13]
```diff
@@ -62,14 +66,18 @@ Examples of generic search attributes that work across many different data sourc
 Some other attributes are client specific, and are found under client specific submodules, e.g. `attrs.vso <sunpy.net.vso.attrs>` and `attrs.jsoc <sunpy.net.jsoc.attrs>`.
 The full list of attributes can be found in the `attrs submodule reference <sunpy.net.attrs>`.
 
-Some search attributes need one or more values specifying, for example ``Time`` needs at least a start and an end date to specify a time range::
+Some search attributes need one or more values specifying, for example ``Time`` needs at least a start and an end date to specify a time range:
+
+.. code-block:: python
 
     >>> a.Time('2012/3/4', '2012/3/6')
     <sunpy.net.attrs.Time(2012-03-04 00:00:00.000, 2012-03-06 00:00:00.000)>
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:75]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 14]
```diff
@@ -62,14 +66,18 @@ Examples of generic search attributes that work across many different data sourc
...
 
 For attributes that can take a range of different values, printing the attribute lists the values sunpy knows about.
 These values are updated with every release of sunpy, so may not be quite up to date!
-As an example::
+As an example:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:80]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 15]
```diff
@@ -116,7 +128,9 @@ To see a summary of the results print the result variable that came back from th
     <BLANKLINE>
 
 Queries can be made more flexible or specific by adding more attrs objects to the `~sunpy.net.Fido` search.
-As an example, specific passbands can be searched for by supplying an `~astropy.units.Quantity` to the `a.Wavelength <sunpy.net.attrs.Wavelength>` attribute::
+As an example, specific passbands can be searched for by supplying an `~astropy.units.Quantity` to the `a.Wavelength <sunpy.net.attrs.Wavelength>` attribute:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:133]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 16]
```diff
@@ -136,7 +150,9 @@ As an example, specific passbands can be searched for by supplying an `~astropy.
     <BLANKLINE>
     <BLANKLINE>
 
-Data of a given cadence can also be specified using the `a.Sample <sunpy.net.attrs.Sample>` attribute::
+Data of a given cadence can also be specified using the `a.Sample <sunpy.net.attrs.Sample>` attribute:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:155]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst (hunk 8 above)
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:241]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 17]
```diff
@@ -262,7 +297,9 @@ If you then wanted to inspect just the LYRA data for the whole time range specif
     2012-01-01 00:00:00.000 2012-01-01 23:59:59.999       LYRA ...      ESA     2
     2012-01-02 00:00:00.000 2012-01-02 23:59:59.999       LYRA ...      ESA     2
 
-Or, equivalently::
+Or, equivalently:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:302]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 18]
```diff
@@ -280,86 +317,74 @@ As we have seen above the `~sunpy.net.fido_factory.UnifiedResponse` object conta
 Each of the responses are `~sunpy.net.base_client.QueryResponseTable` objects, which are `astropy.table` objects meaning that you can interact with them and filter them like any other tabular data.
 This can be used to interact with results which are metadata only, i.e. searches from the HEK, or it can be used to reduce the number of files downloaded by `Fido.fetch <sunpy.net.fido_factory.UnifiedDownloaderFactory.fetch>`.
 
-For example if we did a query for some AIA and HMI data::
+For example if we did a query for some AIA and HMI data:
 
-    >>> results = Fido.search(a.Time("2020/01/01", "2020/01/01 00:05"), a.Instrument.aia | a.Instrument.hmi)  # doctest: +REMOTE_DATA
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:322]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 19]
```diff
@@ -397,35 +422,21 @@ And then we can pick which ones to see with the :meth:`~.UnifiedResponse.show` m
...
     <BLANKLINE>
     <BLANKLINE>
 
 To give an example of filtering post-search, let's only return the rows in the table which are line-of-sight magnetograms from HMI or the 94Å passband from AIA.
 You can also always do this filtering with the `a.Physobs <sunpy.net.attrs.Physobs>` and `a.Wavelength <sunpy.net.attrs.Wavelength>` attrs in the search command.
 
-First we split the results in to a table for AIA and a table for HMI::
+First we split the results in to a table for AIA and a table for HMI:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:433]: - The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 20]
```diff
@@ -434,14 +445,10 @@ We can use boolean indexing to match the value of the ``"Physobs"`` column::
...
-    2020-01-01 00:01:52.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
-    2020-01-01 00:02:37.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
-    2020-01-01 00:03:22.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
-    2020-01-01 00:04:07.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
-    2020-01-01 00:04:52.000        HMI 6173.0 .. 6174.0 LOS_magnetic_field
 
-To match the ``"Wavelength"`` column we need to account for the fact that VSO results return a wavelength range of ``[min, max]`` so we match the min::
+To match the ``"Wavelength"`` column we need to account for the fact that VSO results return a wavelength range of ``[min, max]`` so we match the min:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:451]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 21]
```diff
@@ -454,36 +461,20 @@ To match the ``"Wavelength"`` column we need to account for the fact that VSO re
...
-
 .. _downloading_data:
 
 Downloading data
 ****************
 Once you have located your files via a `Fido.search <sunpy.net.fido_factory.UnifiedDownloaderFactory.search>`, you can download them via `Fido.fetch <sunpy.net.fido_factory.UnifiedDownloaderFactory.fetch>`.
-Here we'll just download the first file in the result::
+Here we'll just download the first file in the result:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:477]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 22]
```diff
@@ -493,29 +484,39 @@ Here we'll just download the first file in the result::
...
 
   >>> downloaded_files = Fido.fetch(results, path='/ThisIs/MyPath/to/Data/{file}')  # doctest: +SKIP
 
 This downloads the query results into the directory ``/ThisIs/MyPath/to/Data``, naming each downloaded file with the filename ``{file}`` obtained from the client.
 You can also use other properties of the returned query to define the path where the data is saved.
-For example, to save the data to a subdirectory named after the instrument, use::
+For example, to save the data to a subdirectory named after the instrument, use:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:497]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 23]
```diff
@@ -493,29 +484,39 @@ Here we'll just download the first file in the result::
...
 
     >>> downloaded_files = Fido.fetch(results, path='./{instrument}/{file}')  # doctest: +SKIP
 
-You can see the list of options that can be specified in path for all the files to be downloaded with ``results.path_format_keys``::
+You can see the list of options that can be specified in path for all the files to be downloaded with ``results.path_format_keys``:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:503]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

File: docs/tutorial/acquiring_data/fido.rst [hunk 24]
```diff
@@ -493,29 +484,39 @@ Here we'll just download the first file in the result::
...
-    ['end_time', 'extent_length', 'extent_type', 'extent_width', 'fileid', 'info', 'instrument', 'physobs', 'provider', 'size', 'source', 'start_time', 'wavelength', 'wavetype']
+    ['end_time', 'extent_length', 'extent_type', 'extent_width', 'fileid', 'instrument', 'physobs', 'provider', 'size', 'source', 'start_time', 'wavelength', 'wavetype']
 
 Retrying Downloads
 ^^^^^^^^^^^^^^^^^^
 If any files failed to download, the progress bar will show an incomplete number of files (i.e. 100/150) and the `parfive.Results` object will contain a list of the URLs that failed to transfer and the error associated with them.
-This can be accessed with the ``.errors`` attribute or by printing the `~parfive.Results` object::
+This can be accessed with the ``.errors`` attribute or by printing the `~parfive.Results` object:
+
+.. code-block:: python
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply docs/tutorial/acquiring_data/fido.rst:513]: The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.

**Pull request merged**